zulipterminal/config/keys.py

```diff
@@ -305,6 +305,11 @@
         'help_text': 'Clear compose screen',
         'key_category': 'msg_compose',
     }),
+    ('USER_INFO', {
+        'keys': ['u'],
+        'help_text': 'View user description',
+        'key_category': 'msg_actions',
```

Comment:
I totally feel you :) Just like the edit history view in the message info, I'll add the key for the user info view (`i`) so that we're clear with the keys.
Yeah, I'll hook the README and keys commit together :)